applications/ananke/src/ananke_callback_sup.erl

```diff
@@ -1,88 +0,0 @@
-%%%-------------------------------------------------------------------
```

Comment:
@k-anderson yes this is moved to `ananke_tasks_sup`. vm_callblack should be working as before but it needs refactoring and code clean up anyway in another PR maybe. I fixed some broken code there though.